node/libs/roles/src/attester/messages/batch.rs

```diff
@@ -106,7 +115,8 @@ impl BatchQC {
         let attesters = genesis
             .attesters
             .as_ref()
-            .expect("attesters set is empty in genesis"); // This case should never happen
+            .context("no attester committee in genesis")
```

Comment:
this line doesn't make sense, especially since you discard the error. use Option::ok_or/ok_or_else